src/_data/side-nav.yml

```diff
@@ -82,10 +58,21 @@
           match-page-url-exactly: true
           permalink: /null-safety/faq
 
-    - title: Evolution
-      permalink: /guides/language/evolution
-    - title: Specification
-      permalink: /guides/language/spec
+- title: Effective Dart
```

Comment:
**praise:** This is _great_! I think it will encourage readers to use this section more.

**question:** How do you feel this works with the **Development** section? Lots of cross links?

Reply:
Yeah it's great to have a top-level entry for Effective Dart. It is pretty standalone and has some references to language docs, but not too many internal references back to it (outside of the linter rules).

Reply:
> question: How do you feel this works with the Development section? Lots of cross links? 

I'm not sure what you mean by that. The only pages I actually moved in the file directory are `code-of-conduct`, `dart-2` (old migration guide), and `dartpad-best-practices`, all reflected in the redirect file. No links are broken by rearranging the side nav, if that's what you meant